### DISCUSSION TITLE
Local account creation

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3535] [Error / Bug Report]
**Describe the bug**
Local account creation with the email address matching an email from your LDAP user. The LDAP synchronization doesn't work.

**To Reproduce**
- Start a new instance of WIKI.JS
- Use an email matching the administrator from your AD user.
- Use that account to configuration LDAP connection.

**Expected behavior**
I expected the synchronization to work. 

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04, AWS image.
 - Wiki.js version: 2.4.X
 - Database engine: postgres 9.7

**Additional context**
Not sure why it is relevant to force the use of email addresses to create _local_ account. It is somewhat confusing at first when you install the WIKI.

**Comment by winkelement** [comment]
I'm sorry i have read your description several times but cannot figure out **what exactly is the bug**? 😕

**Comment by ghost** [comment]
Thanks for pointing that out, I now see that I haven't wrote the issue haha. 

The LDAP authentication fails. From my understanding, due to the local admin matching some LDAP fields. When I changed the Admin Binding credentials to something completely different, it worked.

**Comment by winkelement** [comment]
Ah ok now i understand. Yeah i remember now i had the same issue and it took me a while to figure that out. 
So before thinking about if and/or how to adress this issue i guess it would be helpful to add that info to the docs at https://docs.requarks.io/en/auth/ldap

**Comment by ghost** [comment]
I guess too, but I am wondering if not forcing a default username/password wouldn't be easier to avoid this issue? 

For instance, if from the start, you do not create the account, but you do go through the configuration process, it might avoid this sort of mistake/misunderstanding. And then, after you login the first time, it requires you to change the password.
